src/Microsoft.ML.Data/Transforms/HashTransform.cs

```diff
@@ -541,6 +572,8 @@ private ValueGetter<VBuffer<uint>> ComposeGetterVec(IRow input, int iinfo)
             var seed = ex.HashSeed;
             var len = Infos[iinfo].TypeSrc.VectorSize;
             var src = default(VBuffer<T>);
+            T[] denseValues = null;
+            int expectedSrcLength = Infos[iinfo].TypeSrc.VectorSize;
 
             if (!ex.Ordered)
```

Comment:
Does the presence of these conditions not compromise your strategy of using `null` on `hasherSparse` as a signal?